apps/examples/testcase/le_tc/kernel/tc_sched.c

```diff
@@ -135,6 +159,23 @@ static void tc_sched_sched_setget_scheduler_param(void)
 	int arr_idx = 0;
 	int sched_arr[ARRLEN] = { SCHED_RR, SCHED_FIFO };
 
+	/*  Check null/invalid priority parameter */
+
+	ret_chk = sched_setparam(getpid(), NULL);
+	TC_ASSERT_EQ("sched_setparam", ret_chk, ERROR);
+
+	ret_chk = sched_getparam(getpid(), NULL);
+	TC_ASSERT_EQ("sched_getparam", ret_chk, ERROR);
+
+	ret_chk = sched_setscheduler(0, SCHED_RR, &st_setparam);
+	TC_ASSERT_EQ("sched_setscheduler", ret_chk, ERROR);
```

Comment:
sched_setscheduler will return ERROR when priority is not in valid range. So I think you'd better set st_setparam explicitly for readability.